src/data.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    order:{
```

Comment:
A valid .json file needs each attribute name value pair to be escaped with double quotes.

### Correct
```javascript
{
  "order": {}
}
```

### Wrong

```javascript
{
  order: {}
}
```